### ISSUE TITLE
Variable as Window or Door?

### ISSUE THREAD

**Issue by robbyHuelsi** [thkl/homebridge-homematic#95]
Is it possible to change a variable in my CCU2 to a window or a door?
I tried to add the name of the variable to the window/door list in my config.json, but it seems that's not the right way. ^^ Is there another solution?

**Comment by thkl** [comment]
you can add a boolean variable to Homekit. Just put the name of that variable into the section "variables" of your config.json

**Comment by robbyHuelsi** [comment]
Thanks for your answer. I think I described my problem not clear enough. 😕

Sure, adding variables to section "variables" works well! But in the Home app I only able to choose fan, light or switch as a type of this new "device". My problem is, that I didn't find a solution to change the type to window, door or garage opener.
In my case the variable represents a self-made garage opener and I want to say to Siri "Open garage door" instead of "Turn garage door on". I know, thats just a small difference, but that makes harder to use for non-nerds. 😉 

I tried to add the name of the variable to the window/door list in my config.json, but it seems that's not the right way. Is there another solution?
